custom/icds_reports/utils/aggregation_helpers/distributed/agg_dashboard_activity.py

```diff
@@ -143,12 +143,12 @@ def get_last_agg_date(self):
         from custom.icds_reports.models.aggregate import DashboardUserActivityReport
 
         #because we dont expect the report to fail for 7 consecutive days
-        seven_days_back = self.date - timedelta(days=1)
+        seven_days_back = self.date - timedelta(days=7)
 
         result = (DashboardUserActivityReport.objects.
                   filter(date__lt=self.date.strftime("%Y-%m-%d"), date__gt=seven_days_back).
                   aggregate(Max('date')))
-        if result:
+        if result.get('date__max'):
```

Comment:
this (and the following lines) can be simplified to `return result.get('date__max', date(1970, 1, 1))`

Reply:
ahh yeah.